src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApiSearchTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.ServiceConfigItem;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.config.ServiceConfigProvider;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.SearchResult;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class CcdApiSearchTest {
+
+    @Mock CoreCaseDataApi feignCcdApi;
+    @Mock CcdAuthenticatorFactory authenticatorFactory;
+    @Mock ServiceConfigProvider serviceConfigProvider;
+
+    @Mock CcdAuthenticator ccdAuthenticator;
+
+    ArgumentCaptor<String> idamTokenCaptor = ArgumentCaptor.forClass(String.class);
+    ArgumentCaptor<String> s2sTokenCaptor = ArgumentCaptor.forClass(String.class);
+    ArgumentCaptor<String> caseTypeCaptor = ArgumentCaptor.forClass(String.class);
+    ArgumentCaptor<String> searchQueryCaptor = ArgumentCaptor.forClass(String.class);
```

Comment:
I'd put these captors in the test, as they seem to belong to it. Up to you.

Reply:
Before this test class were added there were not unit tests for CcdApi.
Hopefully more test will be added to this class soon and those captors will be used as well.